### ISSUE TITLE
Exception when clicking in file with tests

### ISSUE THREAD

**Issue by Gert-dev** [yacut/tester#19]
Hello

This issue seems to happen every time I open a file that contains tests. I'm using tester-phpunit and version 1.3.1 of the package. If this is something that tester-phpunit is doing wrong, I'll report it there instead :-).

```
/home/user/.atom/packages/tester/node_modules/rxjs/observable/PromiseObservable.js:76
Hide Stack Trace
Error: Actions must be plain objects. Use custom middleware for async actions.
    at dispatch (/home/user/.atom/packages/tester/node_modules/redux/lib/createStore.js:166:13)
    at /home/user/.atom/packages/tester/node_modules/redux-observable/lib/cjs/createEpicMiddleware.js:71:22
    at SafeSubscriber.dispatch [as _next] (/home/user/.atom/packages/tester/node_modules/redux/lib/applyMiddleware.js:45:18)
    at SafeSubscriber.__tryOrUnsub (/home/user/.atom/packages/tester/node_modules/rxjs/Subscriber.js:238:16)
    at SafeSubscriber.next (/home/user/.atom/packages/tester/node_modules/rxjs/Subscriber.js:185:22)
    at Subscriber._next (/home/user/.atom/packages/tester/node_modules/rxjs/Subscriber.js:125:26)
    at Subscriber.next (/home/user/.atom/packages/tester/node_modules/rxjs/Subscriber.js:89:18)
    at SwitchMapSubscriber.notifyNext (/home/user/.atom/packages/tester/node_modules/rxjs/operator/switchMap.js:124:30)
    at InnerSubscriber._next (/home/user/.atom/packages/tester/node_modules/rxjs/InnerSubscriber.js:23:21)
    at InnerSubscriber.Subscriber.next (/home/user/.atom/packages/tester/node_modules/rxjs/Subscriber.js:89:18)
    at MergeAllSubscriber.OuterSubscriber.notifyNext (/home/user/.atom/packages/tester/node_modules/rxjs/OuterSubscriber.js:19:26)
    at InnerSubscriber._next (/home/user/.atom/packages/tester/node_modules/rxjs/InnerSubscriber.js:23:21)
    at InnerSubscriber.Subscriber.next (/home/user/.atom/packages/tester/node_modules/rxjs/Subscriber.js:89:18)
    at CatchSubscriber.Subscriber._next (/home/user/.atom/packages/tester/node_modules/rxjs/Subscriber.js:125:26)
    at CatchSubscriber.Subscriber.next (/home/user/.atom/packages/tester/node_modules/rxjs/Subscriber.js:89:18)
    at SwitchMapSubscriber.notifyNext (/home/user/.atom/packages/tester/node_modules/rxjs/operator/switchMap.js:124:30)
    at InnerSubscriber._next (/home/user/.atom/packages/tester/node_modules/rxjs/InnerSubscriber.js:23:21)
    at InnerSubscriber.Subscriber.next (/home/user/.atom/packages/tester/node_modules/rxjs/Subscriber.js:89:18)
    at /home/user/.atom/packages/tester/node_modules/rxjs/observable/PromiseObservable.js:66:36
```

Thanks in advance

**Comment by yacut** [comment]
Unfortunately I can not reproduce the problem. I used https://github.com/recca0120/tester-phpunit project.
Could you please give more information: command, test project etc?

**Comment by Gert-dev** [comment]
I'm experiencing it with [my project](https://gitlab.com/php-integrator/core) on the development branch as soon as I open or activate a file that contains tests. It only happens once, then everything ceases to work. Project testing works as long as I don't open any test file.

I tried stepping through with a debugger and it seems to happen in `tester/node_modules/rxjs/src/observable/PromiseObservable.ts` right on the line `subscriber.next(value);` (line 66 here). As soon as this line is executed, the `.then` clause is triggered that throws the error. The `value` that is passed here is also `undefined`, not sure if that is of any help to you.

**Comment by yacut** [comment]
Unfortunately I can still not reproduce the problem.  MacOS 10.12.5/ PHP 7.1.5 / Tester 1.3.1
<img width="1440" alt="screen shot 2017-06-22 at 21 54 11" src="https://user-images.githubusercontent.com/4711633/27453343-5e50aa8e-5796-11e7-9787-ee0622559389.png">

**Comment by recca0120** [comment]
Windows 10/ PHP 7.1.1 / Tester 1.3.2
![1](https://user-images.githubusercontent.com/1390554/27463682-b7b33dc0-57fb-11e7-9c62-8309fbacef05.png)

**Comment by Gert-dev** [comment]
Very strange, I think it's now fixed by pulling 1.3.2 from Git and running `apm install` manually. I'll reopen if I encounter this again.
